code/parachain/frame/fnft/README.md

```diff
@@ -1,4 +1,27 @@
-# Overview
+# fNFT
+This pallet allows the creation of financial NFTs(fNFTs) with specific attributes and provides abstractions for 
+typed and reference NFT design.
+
+---
+
+## Overview
+
+Pallet fNFT provides an implementation of NFTs as proxied accounts able to take ownership of assets and create 
```

Comment:
We can link to the RFC/Design doc for further reference
https://github.com/ComposableFi/composable/blob/main/rfcs/0006-financial-nft.md